DiscordBot.Modules/CommandModules/PingModule.cs

```diff
@@ -1,23 +1,14 @@
 ﻿using System.Threading.Tasks;
 using Discord.Commands;
-using Microsoft.Extensions.Logging;
 
 namespace DiscordBot.Modules.CommandModules
 {
     public class PingModule : ModuleBase
     {
-        private readonly ILogger<PingModule> logger;
```

Comment:
Why did u remove the logger?

Reply:
Why do we use it in PingModule, when we don‘t use it in many other commands?

Reply:
Logging is good practice. Not using it in other code is no valid justification for enforcing it in the whole codebase.